app/src/main/java/com/framgia/fsalon/utils/binding/BindingUtils.java

```diff
@@ -19,14 +18,18 @@
 
 import com.bumptech.glide.Glide;
 import com.framgia.fsalon.R;
+import com.framgia.fsalon.data.model.Service;
 import com.framgia.fsalon.data.model.Stylist;
 import com.framgia.fsalon.screen.booking.BookingViewModel;
+import com.framgia.fsalon.utils.LayoutManager;
 import com.toptoche.searchablespinnerlibrary.SearchableSpinner;
 
 /**
  * Created by MyPC on 20/07/2017.
  */
 public class BindingUtils {
+    private static int VERTICAL_ITEM_SPACE = 5;
```

Comment:
WARNING: [SuspiciousConstantFieldName] The field name indicates a constant but its modifiers do not
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#SuspiciousConstantFieldName